Explorer/Assets/DCL/Roads/Pools/RoadAssetsPool.cs

```diff
@@ -43,7 +81,34 @@ public bool Get(string key, out Transform roadAsset)
         {
             if (roadAssetPoolDictionary.TryGetValue(key, out IObjectPool<Transform> roadAssetPool))
             {
+                if (roadAssetPool.CountInactive == 0)
+                {
+                    int extraInstances = Mathf.Clamp((roadAssetPool as ObjectPool<Transform>).CountAll / 2, POOLS_MIN_NEW_INSTANCES, POOLS_MAX_NEW_INSTANCES);
+                    Transform[] precachedInstances = new Transform[extraInstances];
+
+                    for (int i = 0; i < extraInstances; ++i)
+                    {
+                        precachedInstances[i] = roadAssetPool.Get();
+                    }
+
+                    for (int i = 0; i < extraInstances; ++i)
+                    {
+                        roadAssetPool.Release(precachedInstances[i]);
+                    }
+                }
+
+                // Debug: Uncomment this to know the content of the pools
```

Comment:
Can we use ReportHub here? Similar to what we do [here](https://github.com/decentraland/unity-explorer/blob/ea3c2bfba928338a6f4d509f7e1b63010f6d43b0/Explorer/Assets/DCL/Roads/Systems/RoadInstantiatorSystem.cs#L63), and use `ReportCategory.ROADS` as rport category. In that way, we wont have commented code

Reply:
I wanted to avoid creating the strings and all that stuff, independently from where are the string sent. Does ReportHub provide something for that?

Reply:
Ah, I see. The string will be created nonetheless, `ReportHub` will just filter it if the flag is enabled.

You could use [IsCategoryEnabled](https://github.com/decentraland/unity-explorer/blob/ea3c2bfba928338a6f4d509f7e1b63010f6d43b0/Explorer/Assets/DCL/PerformanceAndDiagnostics/Diagnostics/ReportsHandling/Settings/IReportsHandlingSettings.cs#L19) to see if Roads is enabled and create the string if that condition is met.

Reply:
Its a bit of an overkill though, I dont know how hard would be to inject `IReportsHandlingSettings` here. Do you think its worth it?

Reply:
I don't think it's worth. In the end it's something too specific, only a person who is debugging the pool would be interested in logging that.

Reply:
`ReportHub.Log` is under `Conditional` compilation. If you can provide a single expression (even LINQ) as a string then it will be compiled out from the builds and bring no overhead